.travis.yml

```diff
@@ -38,6 +38,18 @@ jobs:
         - find . -type f -name '*.md' -exec awesome_bot {} \;
       notifications:
         email: false
+    - stage: 'Lint markdown files'
+      language: node_js
+      node_js: 12
+      os: linux
+      before_install: skip
+      install:
+        - npm i -g markdown-spellcheck
+      before_script:
+        - wget --quiet https://gist.githubusercontent.com/juancarlostong/dad02feeebc8763af35b4fdc717cf7a2/raw/29e085302a9418edbefba15d8f790fa76e0a89d5/.spelling
```

Comment:
opting for a simple wget rather than the more expensive cloning of an entire repo. we only need this one file anyway and there's no secrets in it.

Reply:
Does it have to be node? Is there a more performant platform we can use? 

Also, I'd prefer if we can run this in parallel to the other stages so we don't have to wait for it

Reply:
it runs in parallel within the 'Lint markdown files' stage. in practice: 
<img width="1329" alt="Screen Shot 2020-02-07 at 10 36 26 AM" src="https://user-images.githubusercontent.com/40373238/74055944-d2929d00-4995-11ea-8812-d71495716c4f.png">

here you can see it finishes in 39 seconds, which is faster than the other job that ran in parallel (43s) meaning that it does not add to the existing build time